app/data/data-chat/src/main/kotlin/com/hedvig/android/data/chat/di/DataChatModule.kt

```diff
@@ -19,7 +18,6 @@ val dataChatModule = module {
     val applicationContext = get<Context>()
     Room
       .databaseBuilder<AppDatabase>(applicationContext, dbFile.absolutePath)
-      .fallbackToDestructiveMigration(true)
```

Comment:
Not needed if we do in fact provide the right migration plan in annotation with `autoMigrations` over the `AppDatabase` class.